school/diary/templates/diary/achievements_detail.html

```diff
@@ -0,0 +1,27 @@
+{#{% extends "./info_football_player.html" %}#}
+{##}
+{#{% block title %}#}
+{#    <h2>Achievements</h2>#}
+{#{% endblock title %}#}
+{##}
+{#{% block main_part %}#}
+{#    <ul>#}
+{#        {% for a in achievement_set.all %}#}
+{#            <li>{{ a.achievement }}</li>#}
+{#        {% endfor %}#}
+{#    </ul>#}
+{#{% endblock %}#}
+
+{% extends "./info_football_player.html" %}
+
+{% block title %}
+    <h2>Selected Achievement - {{ achiev }}</h2>
```

Comment:
From the view you send a context with the variable `achievement` and using in the template `achiev` so nothing is shown to the user.

```
def achievements_detail(request, achievements_id):
    reaching = Achievement.objects.get(pk=achievements_id)
    return render(request, "diary/achievements_detail.html", {"achievement": reaching})
```

You should send `achiev` as a kye in the view context, or rename `achiev` variable in the template to `achievement`.